test/perf-test/preempt-exp-kwok.yaml

```diff
@@ -38,7 +37,7 @@ spec:
             metadata:
               namespace: default
               labels:
-                appwrapper.mcad.ibm.com: "fake-defaultaw-schd-spec-with-timeout-1"
+                appwrappers.workload.codeflare.dev: "fake-defaultaw-schd-spec-with-timeout-1"
```

Comment:
AFAIK this is not changing with the new api namespace changes. @astefanutti 

https://github.com/project-codeflare/multi-cluster-app-dispatcher/blob/d83fbf21fc342327e1c35fbe5c8e5be270228de2/pkg/apis/controller/v1beta1/appwrapper.go#L29

Reply:
I can revert the label back to `appwrapper.mcad.ibm.com` if that's is what's needed.  It doesn't match the crd then:
```
oc get crd |grep appwrapper
appwrappers.workload.codeflare.dev 
```
But it works fine if I change it back since it's just a label:
```
oc describe pods |grep mcad.ibm.com
Labels:           appwrapper.mcad.ibm.com=defaultaw-schd-spec-with-timeout-1
Labels:           appwrapper.mcad.ibm.com=defaultaw-schd-spec-with-timeout-2
Labels:           appwrapper.mcad.ibm.com=defaultaw-schd-spec-with-timeout-3
Labels:           appwrapper.mcad.ibm.com=defaultaw-schd-spec-with-timeout-4
Labels:           appwrapper.mcad.ibm.com=defaultaw-schd-spec-with-timeout-5
```
With the change, it's actually getting both labels:
```
oc describe pods |grep appwrapper
Labels:           appwrapper.mcad.ibm.com=defaultaw-schd-spec-with-timeout-1
                  appwrappers.workload.codeflare.dev=defaultaw-schd-spec-with-timeout-1
Labels:           appwrapper.mcad.ibm.com=defaultaw-schd-spec-with-timeout-2
                  appwrappers.workload.codeflare.dev=defaultaw-schd-spec-with-timeout-2
Labels:           appwrapper.mcad.ibm.com=defaultaw-schd-spec-with-timeout-3
                  appwrappers.workload.codeflare.dev=defaultaw-schd-spec-with-timeout-3
Labels:           appwrapper.mcad.ibm.com=defaultaw-schd-spec-with-timeout-4
                  appwrappers.workload.codeflare.dev=defaultaw-schd-spec-with-timeout-4
Labels:           appwrapper.mcad.ibm.com=defaultaw-schd-spec-with-timeout-5
                  appwrappers.workload.codeflare.dev=defaultaw-schd-spec-with-timeout-5
```

Reply:
@KPostOffice @jbusche yes that label hasn't been changed as part of the API groups renaming. It may be changed once we've clearly evaluated what's the purpose of that label. For the time being, it can be left as is.